CONTRIBUTING.MD

```diff
@@ -0,0 +1,25 @@
+Contributing
+============
+Thank you for your interest in `tools-python`. The project is open-source software, and bug reports, suggestions, and most especially patches are welcome.
+
+Issues
+------
+`tools-python` has a [project page on GitHub](https://github.com/spdx/tools-python/) where you can [create an issue](https://github.com/spdx/tools-python/issues/new) to report a bug, make a suggestion, or propose a substantial change or improvement that you might like to make. You may also wish to contact the SPDX working group technical team through its mailing list, [spdx-tech@lists.spdx.org](mailto:spdx-tech@lists.spdx.org).
+
+If you would like to work on a fix for any issue, please assign the issue to yourself prior to creating a patch.
+
+Patches
+-------
+The source code for `tools-python` is hosted on [github.com/spdx/tools-python](https://github.com/spdx/tools-python). Please review [open pull requests](https://github.com/spdx/tools-python/pulls) before committing time to a substantial revision. Work along similar lines may already be in progress.
+
+To submit a patch via GitHub, fork the repository, create a topic branch from `master` for your work, and send a pull request when ready. If you would prefer to send a patch or grant access to pull from your own Git repository, please contact the project's contributors by e-mail.
+
+To contribute an implementation of a feature defined by a version of the SPDX specification later than the one supported by the current SPDX Tools release, clone the branch `spec/X.X`, where X.X is the major.minor version of the targeted specification (e.g. "3.0").
```

Comment:
This well intentioned but is overkill IMHO. Best might be to remove that blob

Reply:
This is worth another ticket, but how does `tools-python` deal with parsing different versions of the SPDX spec?